docs/2.0/admin-guide.md

```diff
@@ -909,6 +909,23 @@ $ ssh-keygen -L -f ~/.tsh/keys/localhost/jsmith.cert
                 permit-pty
 ```
 
+### HTTP CONNECT Proxy
+
+Some networks funnel all connections through a proxy server where they can be
+audited and access control rules applied. For these scenarios Teleport supports
+HTTP CONNECT tunneling.
+
+To use HTTP CONNECT tunneling, simply set either the `https_proxy` or
```

Comment:
Yeah, I was confused by that as well, it looks like all software looks for either all upper or all lower. From Go docs:

```
ProxyFromEnvironment returns the URL of the proxy to use for a given request, as indicated by the environment variables HTTP_PROXY, HTTPS_PROXY and NO_PROXY (or the lowercase versions thereof). HTTPS_PROXY takes precedence over HTTP_PROXY for https requests.
```

That being said, I'll make it all upper so our own use of environment variables is consistent.